components/Cards/src/ColorThemer/MDCCardsColorThemer.m

```diff
@@ -28,4 +32,24 @@ + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
   cardCell.backgroundColor = colorScheme.surfaceColor;
 }
 
++ (void)applyStrokeVariantWithColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                                   toCard:(nonnull MDCCard *)card {
+  card.backgroundColor = colorScheme.surfaceColor;
```

Comment:
if we are adding things unrelated to color such as elevation, it seems like MDCCardsColorThemer might not be the best spot for it. How about MDCCardsThemer?

Reply:
Ah - agreed. Let's continue follow the pattern Randall paved with the ButtonThemer types.